caikit/config/config.yml

```diff
@@ -87,10 +87,15 @@ data_streams:
 
 ### Runtime configurations
 runtime:
-    # The runtime library (or libraries) whose models we want to serve using Caikit Runtime. This should
-    # be a snake case string, e.g., caikit_nlp or caikit_cv.
-    library: sample_lib # TODO: replace with libraries below when runtime can support multiple libraries
+    # The runtime library (or libraries) whose models we want to serve using
+    # Caikit Runtime. This should be a snake case string, e.g., caikit_nlp or
+    # caikit_cv.
+    # TODO: replace with libraries below when runtime can support multiple libraries
+    library: sample_lib
     local_models_dir: models
+    # When loading models at boot, wait for the load to complete before moving
+    # on to the next model
```

Comment:
should this say `wait for the load to complete before starting the server`?

Reply:
ah, jk if I'm reading correctly then the loads are currently serial if `wait=True`?

Reply:
oh or not! The `wait` we pass to `self.load_model` is always false, then this `wait` waits for all of them to load

🤯 !